.travis.yml

```diff
@@ -6,3 +6,18 @@ go:
 install:
   - go get github.com/golang/lint/golint
   - go get golang.org/x/tools/cmd/goimports
+
+services:
+  - docker
+
+after_success:
+  - docker login -e $DOCKER_EMAIL -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+  - export REPO=skycirrus/feed-ingress
+  - export TAG=`if [ "$TRAVIS_BRANCH" == "master" ]; then echo "latest"; else echo $TRAVIS_BRANCH ; fi`
+  - cd docker && docker build -f Dockerfile -t $REPO:$COMMIT .
+  - docker tag $REPO:$COMMIT $REPO:$TAG
+  - docker push $REPO
```

Comment:
i haven't found a need for the docker image locally yet, will add release information to the readme